packages/dappmanager/test/modules/ipfs/index.test.int.ts

```diff
@@ -31,11 +26,17 @@ describe("ipfs / integration test", function () {
   });
 
   it("Upload directory", async () => {
-    dirHash = await ipfsAddDirFromFs(dirPath);
+    const addResults = await ipfsAddAll(dirPath);
+    dirHash =
+      addResults
+        .find(addResult => addResult.path === "test_files/ipfs-test-upload")
+        ?.cid.toString() || "";
+    if (!dirHash) throw Error("No directory hash found");
```

Comment:
Abstract this into a separate function, which takes `dirPath` as argument and returns you the CID of the root directory

Reply:
the returns from the `ipfsAdd` and the new `ipfsAddAll` have changed so the function `ipfsAddDirFromFs` did not work as expected